Add language to improve best practice for customer DNS (#495)
18F/federalist.18f.gov: pages/documentation/customer-responsibilities.md

If your domain is not an apex (e.g. 2nd level) domain, the process may be more challenging as some DNS providers do not support all required DNS record types. We recommend that you plan a solution before signing an agreement, please see [custom domains](/documentation/custom-domains) for more details.

##### SPF, DMARC, and MTA-STS records
GSA IT requires that your your URL's apex domain has appropriately set DMARC and SPF records in accordance with [BOD 18-01](https://cyber.dhs.gov/bod/18-01/).

_Expected DMARC record:_
>`v=DMARC1; p=reject; pct=100; fo=1; ri=86400; rua=mailto:dmarcreports@gsa.gov,mailto:reports@dmarc.cyber.dhs.gov; ruf=mailto:dmarcfailures@gsa.gov`

_Expected SPF record:_
>`v=spf1 -all`

> SMTP MTA Strict Transport Security (MTA-STS) is a new standard that can enable domain names to opt into the strict transport layer security mode for email that requires public certificates and encryption. This standard may not be supported by all email providers but we encourage our customers to enable to use this standard if possible and plan on using the domain for email. See the standard ([RFC 3207](https://tools.ietf.org/html/rfc3207)).

Reviewer: ```suggestion
SMTP MTA Strict Transport Security (MTA-STS) is a new standard that can enable domain names to opt into the strict transport layer security mode for email that requires public certificates and encryption. This standard may not be supported by all email providers but we encourage our customers to use this standard if possible when using the domain for email. See the standard ([RFC 3207](https://tools.ietf.org/html/rfc3207)).
```
